src/test/ui/traits/negative-impls/positive-specializes-negative.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(specialization)]
```

Comment:
(Lang+Compiler): I didn't see any test exercising negative specializing negative:
```rust
 #![feature(specialization)]
 #![feature(negative_impls)]

trait MyTrait {}

impl<T> !MyTrait for T {}
impl !MyTrait for u32 {}
```
which seems fine (although not terribly exciting).